scania-digital-design-system/tegel
# feat(popover): close menu programmatically

## **Describe pull-request**  
Added a `close()` method to the popover menu. This will make it possible to programmatically close the menu which is for example needed when having some button as a menu item that doesn't act as a link. 

## **Issue Linking:**  
- [**GitHub**](https://github.com/scania-digital-design-system/tegel/issues/709)

## **How to test**  
_Provide detailed steps for testing, including any necessary setup._
1. Implement for example a button that calls the `close()` method provided by the popover menu.
2. Click the button and make sure that the menu closes.

## **Checklist before submission**

## **Suggested test steps**
    - Not exactly, I wanted an `open` prop instead to align with for example side menu dropdown. But the popover menu has a conflicting variable called `show` that has a slightly other purpose here.
